examples/carousel/css/carousel-tablist.css

```
@@ -0,0 +1,236 @@
+  background-color: transparent;
+  border: none;
+  outline: none;
+  top: 0;
+  left: 6px;
+  z-index: 10;
+}
+
+.carousel-tablist .controls button svg rect.background {
+  stroke: black;
+  fill: black;
+  stroke-width: 1px;
+  opacity: 0.6;
+}
+
+.carousel-tablist .controls button svg rect.border {
+  fill: transparent;
+  stroke: transparent;
+  stroke-width: 2px;
+}
+
+.carousel-tablist .controls button.rotation svg polygon.pause {
+  stroke-width: 4;
+  fill: transparent;
+  stroke: transparent;
+}
+
+.carousel-tablist .controls button.rotation svg polygon.play {
+  stroke-width: 1;
+  fill: transparent;
+  stroke: transparent;
+}
+
+.carousel-tablist .controls button.rotation.pause svg polygon.pause,
+.carousel-tablist .controls button.rotation.play svg polygon.play {
+  fill: white;
+  stroke: white;
+}
+
+.carousel-tablist .controls button.rotation:focus rect.background,
+.carousel-tablist .controls button.rotation:hover rect.background,
+.carousel-tablist .controls button.rotation:focus rect.border,
+.carousel-tablist .controls button.rotation:hover rect.border {
+  fill: #005a9c;
+  stroke: #005a9c;
+  opacity: 1;
+}
+
+.carousel-tablist .controls button.rotation:focus rect.border {
+  stroke: white;
+}
+
+/* Shared CSS for Tabs */
+
+.carousel-tablist [role="tablist"] {
+  position: relative;
+  top: -2px;
+  text-align: center;
+}
+
+.carousel-tablist [role="tablist"] .border {
+  display: inline-block;
+  border: 0 transparent solid;
+  border-radius: 13px;
+  padding-top: 2px;
+  height: 30px;
+  background-color: rgba(0, 0, 0, 0.2);
+}
+
+.carousel-tablist [role="tablist"] .border.focus {
+  border-width: 2px;
+  border-color: white;
+  padding-top: 0;
+  background-color: #005a9c;
+  height: 26px;
+}
+
+.carousel-tablist [role="tab"] {
+  position: relative;
+  top: -2px;
+  padding: 0;
+  margin: 0;
+  background-color: transparent;
+  border: none;
+  outline: none;
+  width: 34px;
+}
+
+.carousel-tablist [role="tab"] circle.border {
+  display: none;
+  z-index: 12;
+}
+
+.carousel-tablist [role="tab"] circle.tab {
+  z-index: 16;
+}
+
+.carousel-tablist [role="tab"] circle.tab-background {
+  stroke: black;
+  fill: black;
+  stroke-width: 2px;
+  opacity: 0.6;
+  z-index: 14;
+}
+
+.carousel-tablist [role="tab"] circle.tab,
+.carousel-tablist [role="tab"] circle.border {
+  stroke: white;
+  fill: transparent;
+  stroke-width: 2px;
+}
+
+.carousel-tablist [role="tab"][aria-selected="true"] circle.tab {
+  fill: white;
+}
+
+.carousel-tablist [role="tab"]:focus circle.border {
+  display: block;
+  fill: #005a9c;
+  stroke: #005a9c;
+}
+
+.carousel-tablist [role="tablist"] .focus circle.tab-background {
+  stroke: #005a9c;
+  fill: #005a9c;
+}
+
+.carousel-tablist [role="tab"]:hover circle.tab-background {
+  fill: white;
+  stroke: white;
+  opacity: 1;
+}
+
+.carousel-tablist [role="tab"]:hover circle.border,
+.carousel-tablist [role="tab"]:hover circle.tab {
+  fill: #005a9c;
+  stroke: #005a9c;
+  opacity: 1;
+}
```

When tablist has focus and _mouse_ is used to select another tab, change circle.tab color to "hovered and selected" color (i.e. white center)

```suggestion
}

.carousel-tablist [role="tab"][aria-selected="true"]:hover circle.tab {
  fill: white;
}
```